pyjs9/__init__.py

```diff
@@ -61,10 +64,14 @@
 
 # load socket.io, if available
 try:
-    from socketIO_client import SocketIO, LoggingNamespace
+#    from socketIO_client import SocketIO, LoggingNamespace
+    import socketio
+
+    print('set socketio transport')
```

Comment:
> Also, did you mean to leave in the print statements detailing which transport mechanism was used? Is this standard Python practice?

@ericmandel  Sorry, my fault. It is not a good Python practice.  
We could use `logging` module instead of print for debugging.  
Use `logging.info('set socketio transport')` instead of `print('set socketio transport')`